doc/modules/pipeline.rst

```diff
@@ -266,4 +269,60 @@ and ignored by setting to ``None``::
 .. topic:: Examples:
 
  * :ref:`sphx_glr_auto_examples_feature_stacker.py`
- * :ref:`sphx_glr_auto_examples_hetero_feature_union.py`
+
+
+.. _column_transformer:
+
+ColumnTransformer for heterogeneous data
+========================================
+
```

Comment:
Maybe a warning here saying that the ColumnTransformer is experimental and that we will change the API.